pkg/kubelet/kubelet.go

```diff
@@ -2845,6 +2854,110 @@ func (kl *Kubelet) fastStatusUpdateOnce() {
 	}, 100*time.Millisecond, stopCh)
 }
 
+// handleMaxContainerCheckpoints goes through the list of checkpoints of a
+// container in a pod and deletes old checkpoints if there are more than
+// 'kl.maxContainerCheckpoints'.
+func (kl *Kubelet) handleMaxContainerCheckpoints(
+	podUID types.UID,
+	podFullName,
+	containerName string,
+	options *runtimeapi.CheckpointContainerRequest,
+) error {
+
+	if kl.maxContainerCheckpoints < 1 {
+		// This cannot really happen as the kubelet config parser will
+		// error out early if this is < 1. This additional error
+		// handling is mainly here to make sure tests are working.
```

Comment:
nit: The check in the pkg/kubelet/apis/config/validation/validation.go is a bit different. It would be good to synchronise this check and that one.

Reply:
please, change the comment as well, as suggested